CHANGELOG.md

```diff
@@ -6,17 +6,56 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [3.0.0] - 2022-03-30
+Amazon Chime SDK JS 3.0.0 is here !! 🎉🎉🎉
+
+Amazon Chime SDK JS 3.0.0 includes the following major changes:
+
+- Each user’s audio and video devices are now fully decoupled from their connection to the WebRTC media session. 
+  A user can select their preferred devices in a preview window prior to joining the session, and the devices do not 
+  have to be programmatically reselected when joining the session. Once connected to the session, the user can switch devices instantly and without interrupting their WebRTC connection.
+- The 3.0 client libraries now use the standardized WebRTC metrics for all browsers.
+- Add support for aws-sdk js v3 for messaging session to help avoid using multiple versions in a single application 
+  and minimize application footprint.
+- Deprecated support for Safari 12, and Plan B in Session Description Protocol (SDP) negotiations  
```

Comment:
1st and 2nd = Complete sentence
3rd = Add (verb)
4th = Deprecated (past tense verb)

Make all points consistent.